velox/substrait/VeloxSubstraitSignature.cpp

```diff
@@ -59,6 +59,58 @@ std::string VeloxSubstraitSignature::toSubstraitSignature(const TypePtr& type) {
   }
 }
 
+// static
+TypePtr VeloxSubstraitSignature::fromSubstraitSignature(
```

Comment:
Locally we implemented a type mapping from Substrait type name to Velox type name as [SubstraitParser.h](https://github.com/oap-project/velox/blob/main/velox/substrait/SubstraitParser.h#L139-L155) shows, and with this function [TypeUtils.cpp](https://github.com/facebookincubator/velox/blob/main/velox/substrait/TypeUtils.cpp#L68), the string can be converted to Velox type.

Reply:
@rui-mo Rui, thank you for the pointers. Would you be interested in contributing these changes?

Reply:
Sure. I expect to open the PR next week, and will ping you for review. Thanks.

Reply:
Hi @mbasmanova, I opened https://github.com/facebookincubator/velox/pull/7181 for the type conversion part. Would you like to take a review? Thanks.